### PR TITLE
Add probes to Fulcio deployment

### PR DESCRIPTION
**Pull request by codysoyland**
## Description of the change

This adds `readinessProbe` and `livenessProbe` to the Fulcio deployment.

Fulcio doesn't have health endpoints, so I'm using `/api/v1/rootCert`.

## Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [x] JSON Schema generated.
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/fulcio/values.schema.json [hunk 1]
```diff
@@ -445,6 +445,14 @@
                         }
                     ]
                 },
+                "livenessProbe": {
+                    "description": "Liveness probe configuration",
+                    "$ref": "https://kubernetesjsonschema.dev/v1.18.1/_definitions.json#/definitions/io.k8s.api.core.v1.Probe"
+                },
+                "readinessProbe": {
+                    "description": "Readiness probe configuration",
+                    "$ref": "https://kubernetesjsonschema.dev/v1.18.1/_definitions.json#/definitions/io.k8s.api.core.v1.Probe"
```
Comment (codysoyland) [inline charts/fulcio/values.schema.json:454]: Any thoughts on using these references to https://kubernetesjsonschema.dev? Not sure what implications this has, as I don't see any `$ref` references within the Sigstore Helm charts.

**Review by bobcallaway — APPROVED** [review]

File: charts/fulcio/values.schema.json (hunk 1 above)
Comment (hectorj2f) [reply charts/fulcio/values.schema.json:473]: How did you generate these values ?

**Comment by bobcallaway** [comment]
FYI @codysoyland https://github.com/sigstore/fulcio/pull/1258

**Comment by bobcallaway** [comment]
fixed by https://github.com/sigstore/helm-charts/pull/568

**Pull request closed without merging**